public/locales/ru/translation.json

```diff
@@ -0,0 +1,63 @@
+{
+  "about": "О сайте",
+  "available": "Доступно",
+  "blog": "Описание работы сайта",
+  "community": "Сообщество",
+  "contributors": "Контрибьюторы",
+  "countryCode": "ru",
+  "title": "Выберите новое уникальное имя для Вашего проекта",
+  "description": "Проверьте доступность идей для названия Вашего нового приложения/библиотеки/организации сразу во всех самых крупных регистрах.",
+  "gettingStarted": "Начать",
+  "gettingStartedWithExample": "Попробовать сначала с namae",
+  "issues": "Issues",
+  "language": "Язык",
+  "noResult": "Результаты отсутствуют",
+  "placeholder": "Напишите сюда желаемое имя",
+  "pressEnterToSearch": "Нажмите Enter, чтобы начать поиск",
+  "providers": {
+    "appStore": "App Store",
+    "archlinux": "Arch Linux",
+    "ubuntu": "Ubuntu",
+    "debian": "Debian",
+    "domains": "Домены",
+    "firebase": "Firebase",
+    "github": "Организация GitHub",
+    "githubSearch": "Репозиторий GitHub",
+    "gitlab": "GitLab",
+    "google": "Google Search",
+    "heroku": "Heroku",
+    "homebrew": "Homebrew",
+    "instagram": "Instagram",
+    "jsorg": "js.org",
+    "modland": "mod.land",
+    "linux": "Linux",
+    "netlify": "Netlify",
+    "now": "Vercel",
+    "npm": "npm",
+    "nta": "Компания (Япония)",
+    "ocaml": "OCaml",
+    "playStore": "Магазин Google Play",
+    "pypi": "PyPI",
+    "rubygems": "RubyGems",
+    "rust": "Rust",
+    "s3": "AWS S3",
+    "slack": "Slack",
+    "spectrum": "Spectrum",
+    "reddit": "Reddit",
+    "twitter": "Twitter",
+    "cloudflare": "Cloudflare Pages",
+    "chromeWebStore": "Chrome Web Store",
+    "firefoxAddons": "Аддоны Firefox",
+    "youtube": "YouTube"
+  },
+  "showMore": "Показать ещё",
+  "try": "Как насчёт этого?",
+  "unavailable": "Недоступно",
+  "uniqueness": {
+    "description": "Шкала УНИК показывает, насколько уникально данное имя (от 0 до 100)",
+    "high": "Довольно-таки уникальное",
+    "low": "Обычное",
```

Comment:
I would maybe use 'Часто используемое' instead (literally translates to "Frequently used")

Reply:
If I will apply this suggestion I would also change moderate uniqueness message in same way

Reply:
Otherwise everything else looks good, thanks!
The translation for "search" still needs more discussion, since it is a pretty big change and would require revisiting all the other localizations. 

Like I told @uetchy i prefer the short version (e.g. "search") because

- It is shorter
- the paragraph under the search bar already explains the usage
- more in line with typical search input placeholder values you find in other websites

Reply:
Thanks for the input. After some discussion I decided we would just go with a short text because of the reasons Daniru described above.
@MadProbe Could you add commits to reflect the suggestions?

Reply:
I have applied the changes also I have resolved merge conflicts